### DISCUSSION TITLE
Do we have a way to specifically filter out test failures only in sf apex run test?

### DISCUSSION THREAD

**Discussion by vctqs1** [forcedotcom/cli#2950] [Q&A] [answered]
Typically, before merging a change, we run the entire test suite. However, as our test suite grows larger, identifying specific test failures becomes challenging.

Do we have any methods to filter out specific test results from the test suite?

**Comment by daveespo** [comment, marked as answer]
It's "coming soon" 

https://github.com/forcedotcom/salesforcedx-apex/issues/243 is addressed via https://github.com/forcedotcom/salesforcedx-apex/pull/385 and https://github.com/salesforcecli/plugin-apex/pull/504

**Reply by mshanemc** [reply]
it'll be in the next nightly

**Comment by VivekMChawla** [comment]
@vctqs1 - Thanks for posting this idea!

We're tracking a similar feature request here: #2872 

I'm going to close this discussion as a duplicate.
